koku/masu/processor/report_processor_base.py

```diff
@@ -38,7 +41,7 @@ class ReportProcessorBase():
     Base object class for downloading cost reports from a cloud provider.
     """
 
-    def __init__(self, schema_name, report_path, compression, provider_id, processed_report):
+    def __init__(self, schema_name, report_path, compression, provider_id, manifest_id, processed_report):
```

Comment:
This constructor is getting a bit big. It might be good to consider using kwargs or passing in an object.